lib/connect_proto/src/substances.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+package primary.connect;
+option go_package = "primary.connect";
+
+import "substance.proto";
+
+message Substances {
```

Comment:
Not needed. See comment on https://github.com/PrimaryDotHealth/connect-proto/pull/15/files#diff-effcd1b13e962633af3828a3942749a57ba7a345b1dbf04be5f4b6c784593fe1R116